### PR TITLE
Vesuvio - LoadVesuvio load monitor property

### LINKED ISSUE #14992: LoadVesuvio Load Monitor algorithm property
Giovanni has asked for an option on `LoadVesuvio` to keep the Monitor workspaces in the ADS after executing the `LoadVesuvio` algorithm.
This should be fairly easy as the monitor data is already loaded.
Tests will also need to be updated for this.

### PR DESCRIPTION
**Pull request by ElliotAOram**
LoadVesuvio algorithm now has the option to load the monitor data in addition to it's normal operation.

**To test:**
- Ensure all the new test cases in LoadVesuvioTest (system test) run and pass.
- Running the following script should produce the expected data from `LoadVesuvio` as well as an additional workspace that contains the monitor data:

```
EVS21000_21005 = LoadVesuvio('21000-21005', '3-134', SumSpectra=True, LoadMonitors=True)
```
- Ensure that there are 2 workspaces produced in the ADS
- They should be named `EVS21000_21005` and `EVS21000_21005_monitors`
- Ensure the `_monitors` workspace contains 2 spectra 
  - When these are plotted they should look like: 
    image

Fixes #14992

[Release Notes](https://github.com/mantidproject/mantid/blob/14992_LoadVesuvio_LoadMonitor_Property/docs/source/release/v3.7.0/indirect_inelastic.rst#id28)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the coding standards? Is it well structured with small focussed classes/methods/functions?
- [x] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [x] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [x] Do changes function as described? Add comments below that describe the tests performed?
- [x] How do the changes handle unexpected situations, e.g. bad input?
- [x] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/PythonInterface/plugins/algorithms/LoadVesuvio.py [hunk 1]
```diff
@@ -126,6 +128,10 @@ def PyInit(self):
                                  "the sum of all of the requested spectra. All detector angles/"
                                  "parameters are averaged over the individual inputs")
 
+        self.declareProperty(LOAD_MON, False,
+                             doc="If true then the monitor data is loaded and will be output by the "
+                                 "algorithm.")
```
Comment (KarlPalmen) [inline Framework/PythonInterface/plugins/algorithms/LoadVesuvio.py]: I'd say "output into a separate workspace by the algorithm".

**Comment by KarlPalmen** [comment]
The release notes have an incorrect link to LoadVesuvio.

**Comment by ElliotAOram** [comment]
@KarlPalmen re. links in the release notes: The links will not work when viewed on gitHub as they need to be built with Sphinx in order to correctly link (which they will be when the release is shipped). So they should be okay

**Comment by KarlPalmen** [comment]
I've checked the release notes in my build and find the link OK there.

**Comment by KarlPalmen** [comment]
I'll test the functionality, built after merge conflicts fixed tomorrow morning.

**Comment by KarlPalmen** [comment]
Functionality tested as suggested works OK :shipit:

**Pull request merged**